4919 queries measurement, 4931 token delays (#4933)
influxdata/docs-v2: content/influxdb/cloud-dedicated/admin/tokens/create.md

The command returns the token ID and the token string.
**This is the only time the token string is available in plain text.**

> InfluxDB might take some time to synchronize new tokens. If a new token doesn't immediately work for querying or writing, wait a few seconds or minutes and then retry your request.

Reviewer: ```suggestion
InfluxDB might take some time to synchronize new tokens.
If a new token doesn't immediately work for querying or writing, wait a few seconds and try again.
```
